### PR TITLE
Improve Node Source permission handling

### PR DESCRIPTION
**Pull request by fviale**
- created new permission for full rm core rights but which does not override node restrictions
 - created a new unused permission to override node restrictions (just in case)
 - created a new group for scheduler full permission
 - "admin" and "demo" users now belong to rmcore and scheduler admin group instead of having "AllPermissions". This allows to enforce node usage restrictions on those accounts
 - improved tests in rm-server/permissions

### CODE REVIEW

File: config/authentication/group.cfg [hunk 1]
```diff
@@ -1,8 +1,10 @@
-admin:admin
-admin:nsadmins
-demo:admin
+admin:scheduleradmins
+admin:rmcoreadmins
+demo:scheduleradmins
+demo:rmcoreadmins
 guest:guests
 nsadmin:nsadmins
+nsadmin2:nsadmins
```
Comment (mbenguig) [inline config/authentication/group.cfg]: Thanks to you, we have now well defined groups: scheduleradmins, rmcoreadmins, nsadmins...  with well defined members: admin, demo... but cant these latter ones replace nsadmin and nsadmins users?

Comment (fviale) [reply config/authentication/group.cfg]: @mbenguig nsadmins users are only allowed to control node sources, not the full resource manager

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/authentication/Client.java [hunk 2]
```diff
@@ -258,12 +259,32 @@ public boolean checkPermission(final Permission permission, String errorMessage)
                 return null;
             }, null);
         } catch (SecurityException ex) {
-            throw new SecurityException(errorMessage, ex);
+            if (!checkSuperUserPermission(superUserPermissions)) {
```
Comment (mbenguig) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/authentication/Client.java]: A single-line comment would have been apreciated ..

Comment (fviale) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/authentication/Client.java]: ok

**Comment by fviale** [comment]
Retest this please

**Pull request merged**